indexer/ai/src/main/java/com/google/cloudsearch/ai/CloudStorageHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.google.cloudsearch.ai;
+
+public class CloudStorageHandler {
+
+    /**
+     * Checks if a given string is a Cloud Storage URI
+     * @param str   string to check
+     * @return  Returns true if the given input is a Cloud Storage URI, else false.
+     */
+    public static boolean isCloudStorageURI(String str) {
+        if(str.substring(0,5).equals("gs://")) {
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
```

Comment:
use startswith and return the boolean result.

Reply:
done